src/test/resources/uk/ac/ebi/embl/api/validation/file/template/TEMPLATE_ERT000002

```diff
@@ -2,7 +2,7 @@
 <name>rRNA gene</name>
 <forms_name>rRNA gene</forms_name>
 <description>For ribosomal RNA genes from prokaryotic, nuclear or organellar DNA. All rRNAs are considered partial.</description>
-<token name="ORGANISM_NAME" type="TAXON_FIELD" mandatory="true" display_name="Organism" description="Full name of organism (generally Genus+species). Formal names should include subspecies (subsp.), variety (var.) and forma (f.) if applicable." tip="Escherichia coli, uncultured bacterium, Bacillus sp. 654g14, Quercus ruber, Saccharomyces cerevisiae, Treponema pallidum subsp. pallidum"/>
+<token name="ORGANISM_NAME" type="TAXON_FIELD" mandatory="true" display_name="Organism" description="Full name of organism (generally Genus+species). Formal names should include subspecies (subsp.), variety (var.) and forma (f.) if applicable. OR sample accession / sample alias / bioSampleId / taxid" tip="Escherichia coli, uncultured bacterium, Bacillus sp. 654g14, Quercus ruber, Saccharomyces cerevisiae, Treponema pallidum subsp. pallidum"/>
```

Comment:
- Remove any files from src/test/resources/uk/ac/ebi/embl/api/validation/file/template/* that are generated during testing and should not be checked in. 
- If this applies to the whole folder, remove all files, write the files into different directory (that is in gitignore) or add this directory to gitignore.
- - Optimally we should create these files in system TEMP directory.